include/usb_cam/usb_cam.hpp

```diff
@@ -221,6 +221,11 @@ class UsbCam
     return m_image.size_in_bytes;
   }
 
+  inline void reset_image_size_in_bytes()
+  {
+    m_image.set_size_in_bytes();
+  }
+
```

Comment:
@hilookas this method should be added to the `image_t` structure, where inside it should set either the height or bytes_per_line variable to 0 and call `set_size_in_bytes()`.